typescript/api/controllers/webservice/RecordController.ts

```diff
@@ -1189,7 +1194,11 @@ export module Controllers {
           }
           let result = await this.RecordsService.updateMeta(brand, oid, record, user);
 
-          return new APIHarvestResponse(harvestId, oid, true, `Record updated successfully`)
+          let updateMessage = "Record updated successfully";
+          if(shouldMerge) {
```

Comment:
Where is `shouldMerge` defined?

Reply:
It's defined at the top of the method
https://github.com/redbox-mint/redbox-portal/blob/356104e9392d82171486f898d8346f401aedb9ab/typescript/api/controllers/webservice/RecordController.ts#L1173

Reply:
Ah, I see. `updateMode` -> `updateHarvestRecord` -> `shouldMerge`